sdk/rust-examples/prime-numbers-generator/generate-set/src/main.rs

```diff
@@ -0,0 +1,45 @@
+//! Prime Numbers Generator: Generate the number set
+//!
+//! ## Context
+//!
+//! Generate all numbers from 2 to the specified upper limit
+//! 
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use std::fs;
+use anyhow;
+
+const OUTPUT_FILENAME: &'static str = "/output/number-set.txt";
+
+fn main() -> anyhow::Result<()> {
+    let mut set: Vec<u32> = Vec::new();
+
+    let args: Vec<String> = std::env::args().collect();
+    let upper_limit = args.get(0)
+        .unwrap()
+        .to_owned()
+        .parse::<u32>()?;
+
+    for i in 2..=upper_limit {
+        set.push(i);
+    }
+
+    let mut set_str: String = set
+        .iter()
```

Comment:
Does this not produce a string with a trailing comma, e.g., `1,2,3,`?  A better way of doing this is to use `.join()` as follows:

```rust
let mut set_str = set.iter().map(|n| n.to_string()).collect::<Vec<_>>().join(",");
```